src/esrs/event_handler.rs

```diff
@@ -0,0 +1,60 @@
+use async_trait::async_trait;
+use uuid::Uuid;
+
+use crate::{AggregateManager, StoreEvent};
+
+#[async_trait]
+pub trait EventHandler<M: AggregateManager>: Send + Sync {
+    // TODO: doc
+    async fn handle(&self, event: &StoreEvent<M::Event>);
```

Comment:
Unlike `TransactionalEventHandler`, this is infallible and does not take an executor to avoid passing an active transaction and we explicitly ignore nested errors, right?

Reply:
Right :)